pages/integrations/sso.md.erb

```diff
@@ -37,7 +37,20 @@ There are two ways to disable a provider:
 1. Using the 'Disable' button in your SSO provider Settings, or
 1. Using the [GraphQL API](/docs/integrations/sso/sso-setup-with-graphql#disabling-an-sso-provider)
 
-If you have switched off all of your SSO providers, users will be required to log in using a username and password. If users don't have a password, and need access while SSO is switched off, they can perform a 'Forgotten Password' reset.
+If you have switched off all of your SSO providers, users will be required to log in using a username and password. If users don't have a password, and need access while SSO is switched off, they can perform a 'Forgotten Password' reset.  
+
+## Migrating from one SSO provider to another SSO provider    
+
+If you are the administrator of an organization within Buildkite with an existing SSO provider set up, and you want to switch to a different SSO provider, these are the steps you need to take:  
+
+1. [Add](/docs/integrations/sso#adding-sso) a new SSO provider, verify it, and allow login from both SSO providers. The users in your organization can continue to sign in and use the same user accounts within Buildkite as long as the emails stay the same.
+2. [Disable and remove](/docs/integrations/sso#disabling-and-removing-sso) the SSO provider you no longer need. If the user credentials (email) stay the same, this is all you need to migrate from one SSO provider to another.  
+
+<div class="Docs__note">
+  <p>If you are also changing the email provider, make sure that Buildkite users in your organization sign in to their existing accounts when performing single sign-on through the new provider. Otherwise, new user accounts will be created, and your organization will be billed for the same users twice.</p>
```

Comment:
Maybe rewording this to something friendlier like 

>  make sure that Buildkite users in your organization sign in to their existing accounts when performing single sign-on through the new provider to prevent your organization being billed twice for the same users

@jayco are we happy to state this in the docs?